src/components/congratulations.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+
+
+  <section class="container"> 
+
+    <div class="left">
```

Comment:
I think bootstrap's left/right classes are used only for carousels. What are you wanting to do here?